src/Medidata.ZipkinTracer.Core/Medidata.ZipkinTracer.Core.csproj

```diff
@@ -58,23 +58,34 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
-    <Reference Include="Thrift, Version=0.9.1.3, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Thrift.0.9.1.3\lib\net35\Thrift.dll</HintPath>
-    </Reference>
   </ItemGroup>
   <ItemGroup>
     <Compile Include="ITraceProvider.cs" />
     <Compile Include="ITracerClient.cs" />
     <Compile Include="IZipkinConfig.cs" />
     <Compile Include="Middlewares\ZipkinMiddleware.cs" />
+    <Compile Include="Models\Annotation.cs" />
+    <Compile Include="Models\AnnotationBase.cs" />
+    <Compile Include="Models\BinaryAnnotation.cs" />
+    <Compile Include="Models\Endpoint.cs" />
+    <Compile Include="Models\References\AnnotationType.cs" />
+    <Compile Include="Models\References\ZipkinConstants.cs" />
+    <Compile Include="Models\Serialization\Json\JsonAnnotation.cs" />
```

Comment:
Duplicate `JSON` in name. `Json/JsonXXXX.cs`
What do you think to merge models under `Json/` into models under `Models/`

```
public class Annotation: AnnotationBase
{
    public DateTimeOffset Timestamp { get; set; } = DateTimeOffset.UtcNow;
    [JsonProperty("value")]
    public string Value { get; set; }
    public int DurationMilliseconds { get; set; }
    [JsonProperty("endpoint")]
    public JsonEndpoint Endpoint => new JsonEndpoint(Host);
    [JsonProperty("timestamp")]
    public long Timestamp => Timestamp.ToUnixTimeMicroseconds();
}
```